tools/replay/parse_dump_resources_cli.cpp

```diff
@@ -78,51 +78,42 @@ static bool CheckIndicesForErrors(const gfxrecon::decode::VulkanReplayOptions& v
     // Indices should be provided sorted
     if (vulkan_replay_options.Draw_Indices.size())
     {
-        bool is_2d = false;
         for (const auto& indices : vulkan_replay_options.Draw_Indices)
         {
-            is_2d |= (indices.size() > 0);
-
             if (!AreIndicesSorted(indices))
             {
-                GFXRECON_LOG_ERROR("ERROR - incomplete --dump-resources parameters");
+                GFXRECON_LOG_ERROR("ERROR - incorrect --dump-resources parameters");
                 GFXRECON_LOG_ERROR("DrawCall indices are not sorted")
                 return true;
             }
         }
-
-        if (!is_2d)
-        {
-            GFXRECON_LOG_ERROR("ERROR - incomplete --dump-resources parameters");
-            GFXRECON_LOG_ERROR("DrawCall indices must be a 2 dimensional array")
-            return true;
-        }
     }
 
     if (vulkan_replay_options.RenderPass_Indices.size())
     {
-        bool is_3d = false;
+        bool is_complete = false;
         for (const auto& indices0 : vulkan_replay_options.RenderPass_Indices)
         {
             if (!indices0.size())
             {
+                is_complete = true;
```

Comment:
Will this allow a `RenderPass` entry that is not 3d?
RenderPass needs to be a 3d array like this:
```
"RenderPass": [
        [[]], [ [ 401, 419 ] ]
    ],
```

Reply:
Fixed in 2d8a134f8a9f952fd31a128e78d2b0b455f1a261